docs/sources/clients/promtail/scraping.md

```diff
@@ -182,7 +182,7 @@ You can relabel default labels via [Relabeling](#relabeling) if required.
 Providing a path to a bookmark is mandatory, it will be used to persist the last event processed and allow
 resuming the target without skipping logs.
 
-see the [configuration](./configuration.md#windows_events) section for more information.
+see the [configuration](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#windows_events) section for more information.
```

Comment:
@KMiller-Grafana Not sure how we should proceed with those link, but previously it would not work on the website.

We could use `relref` short code but this means it doesn't work on github since it's only supported for hugo and not markdown.

In the meantime I've replace it with an absolute link, which might not respect the version sadly.

WDYT ? I don't think we should block the review because of this though, let take care of it later if needed.

Reply:
Ugh, `./configuration.md#windows_events` doesn't work? I wonder if it has to do with `_` (using spaces creates links with `-`). Anyways, it's definitely preferable to use the local links as it allows the site generation tool to create entire sections of the docs for different releases and not always point to `latest`.